infra/batch/datasets/main.py

```diff
@@ -0,0 +1,225 @@
+import base64
```

Comment:
Functions on this script are a core functionality for keeping the dataset information up to date; let's add unit tests.

Reply:
Good point. This should be covered by [this issue](https://github.com/MobilityData/mobility-feed-api/issues/105).